Helper_Scripts/TTS_Installers/install_tts_omnivoice_sidecar.py

```
@@ -0,0 +1,378 @@
+        raise SystemExit(f"Missing prerequisites: {', '.join(missing)}")
+
+
+def clone_repository(repo_url: str, source_dir: Path) -> None:
+    """Clone OmniVoice if the checkout does not already exist."""
+
+    if source_dir.exists():
+        logger.info("Using existing OmniVoice checkout at {}", source_dir)
+        return
+    git_executable = shutil.which("git")
+    if not git_executable:
+        raise SystemExit("Missing prerequisites: git")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)